rational.c

```diff
@@ -2115,10 +2118,16 @@ nilclass_to_r(VALUE self)
 
 /*
  * call-seq:
- *    nil.rationalize([eps])  ->  (0/1)
+ *   rationalize(eps = nil)  ->  (0/1)
+ *
+ * Returns zero as a \Rational:
+ *
+ *   nil.rationalize # => (0/1)
+ *
+ * Argument +eps+ is evaluated:
+ *
+ *   nil.rationalize(fail 'Calamity') # Raises RuntimeError.
```

Comment:
I don't think is surprising behaviour since `eps` is a argument, so it's always evaluated to pass the value to the method. We should just mention that `eps` is ignored.

Reply:
Fixed.